scripts/poclcc/poclcc.c

```diff
@@ -0,0 +1,179 @@
+#include <stdio.h>
```

Comment:
Can you move this to bin/poclcc.c  As this should replace pocl-standalone (the last script we have), it could be removed and the scripts directory removed as well.